Wrote stuff about the name property

maraisr opened the pull request:
Resolves #793

maraisr committed:
docs: wrote stuff about the name property

gr2m commented on docs/guides/configuration.rst:
```diff
@@ -33,6 +33,8 @@ Here is a list of all available options
 | adminPassword   | -                        | ``--adminPassword``      | ``hoodie_adminPassword`` | Password to login to Admin Dashboard. Login is not possible unless set                                              |
 +-----------------+--------------------------+--------------------------+--------------------------+---------------------------------------------------------------------------------------------------------------------+
+| name            | hoodie                   | ``--name``               | ``hoodie_name``          | Name your application.                                                                                              |
++-----------------+--------------------------+--------------------------+--------------------------+---------------------------------------------------------------------------------------------------------------------+
```
the default value is not `hoodie`, but the value of the `"name"` property in the `package.json` file

maraisr replied:
:boom: thanks man - mod going up shortly!

gr2m commented on docs/guides/configuration.rst:
```diff
@@ -54,7 +56,8 @@ default values
         "url": "",
-        "adminPassword": ""
+        "adminPassword": "",
+        "name": "hoodie"
```
maybe use `"my-hoodie-app"` as an example value? `"hoodie"` might be confusing

maraisr replied:
Definitely agree :sunglasses:! Thanks man.

gr2m reviewed (changes requested):
Thank you Marais! I’ve two comments, could you please look into it?

maraisr committed:
chore: code review changes applied

- Config name feels better with a name other than "hoodie".
- The default name is the name property in the `package.json` file.

gr2m reviewed (approved):
Looks great, thanks for the changes 👍
